<launch>

	<node name="speakers_controller_node" pkg="speakers_controller" type="main.py" output="screen" >
	</node>

</launch>